packages/forma-36-codemod/bin/updateDependencies.js

```diff
@@ -0,0 +1,160 @@
+      uninstall: `yarn remove ${pkgs.join(' ')}`,
+    },
+  };
+  return pkgs.length > 0 ? cmds[pkgManager][cmd] : '';
+};
+
+/**
+ * @param {string|string[]} data.pkgManager - name or list of package managers. eg. yarn, npm.
+ * @param {string[]} data.newPackages - List of packages to be installed with the version. eg. '@contentful/f36-components@latest'
+ * @param {string[]} data.removePackages - List of packages to be removed.
+ */
+function getOutput({ newPackages, removePackages, pkgManager = null }) {
+  console.log('You can run the following commands to update the packages:\n');
+
+  // Print commands for each package manager, if no specific one set
+  if (pkgManager === null) {
+    ['yarn', 'npm'].forEach((manager) => {
+      const result = [
+        chalk.bold(`For ${manager}:`),
+        `  ${getCmd(manager, 'uninstall', removePackages)}`,
+        `  ${getCmd(manager, 'install', newPackages)}`,
+      ];
+      console.log(result.filter((r) => r.trim()).join('\n'), '\n');
+    });
+  } else {
+    const result = [
+      chalk.bold(`For ${pkgManager}:`),
+      `  ${getCmd(pkgManager, 'uninstall', removePackages)}`,
+      `  ${getCmd(pkgManager, 'install', newPackages)}`,
+    ];
+    console.log(result.filter((r) => r.trim()).join('\n'), '\n');
+  }
+}
+
+async function updateDependencies(targetDir) {
+  const cwd = path.resolve(process.cwd(), targetDir);
+  const closestPkgJson = readPkgUp.sync({ cwd });
+  let removePackages = packages.PACKAGES_REMOVE;
+  let newPackages = Object.keys(packages.PACKAGES_UPGRADE);
+
+  // If no package.json is found
+  if (!closestPkgJson) {
+    console.log(
+      chalk.yellow(
+        "We didn't find your package.json, you may need to remove or install some packages:",
+      ),
+    );
+
+    newPackages = newPackages.map(
+      (pkg) => `${pkg}@${packages.PACKAGES_UPGRADE[pkg]}`,
+    );
+
+    return getOutput({ newPackages, removePackages });
+  }
+
+  const pkgDirname = path.dirname(closestPkgJson.path);
+  const { packageJson } = closestPkgJson;
+
+  // Merge dependencies property from package.json
+  const allDeps = dependencyProperties.reduce(
+    (acc, dependency) => Object.assign(acc, packageJson[dependency] || {}),
+    {},
+  );
+
+  // Filter packages to remove to only have the ones installed
+  removePackages = removePackages.filter((pkg) => Boolean(allDeps[pkg]));
+
+  // Check if packages version in package.json satisfies the new version
+  newPackages = newPackages
+    .filter(
+      (pkg) =>
+        !allDeps[pkg] ||
+        !semverSatisfies(packages.PACKAGES_UPGRADE[pkg], allDeps[pkg]),
+    )
+    .map((pkg) => `${pkg}@${packages.PACKAGES_UPGRADE[pkg]}`);
+
+  // Select package manager based on lock file, fallback to npm if no lock file is found
+  const pkgManager =
+    (fs.existsSync(path.resolve(pkgDirname, 'yarn.lock')) && 'yarn') ||
+    (fs.existsSync(path.resolve(pkgDirname, 'package-lock.json')) && 'npm') ||
+    'npm';
```

Comment:
Minor 💅
you can probably just leave:
```suggestion
    (fs.existsSync(path.resolve(pkgDirname, 'yarn.lock')) && 'yarn') ||
    'npm';
```